### PR TITLE
6jk9wh: Ticker and test implemented

### PR DESCRIPTION
**Pull request by Sckorn**
Fix 6jk9wh

### CODE REVIEW

File: ticker/include/ticker/Ticker.hpp [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+namespace rcbe::core {
+class Ticker {
+public:
+    using period_type = std::chrono::nanoseconds ;
+    using handler_type = std::function<void(void)>;
+    using clock_type = std::chrono::steady_clock;
+
+    Ticker() = delete;
+    explicit Ticker(std::chrono::milliseconds period);
```
Comment (radameon) [inline ticker/include/ticker/Ticker.hpp:17]: Add more constructors, covering more different time values i.e. seconds, nanosecsonds etc.

File: ticker/include/ticker/Ticker.hpp [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    ~Ticker();
+
+    void set_handler(handler_type&& handler);
```
Comment (radameon) [inline ticker/include/ticker/Ticker.hpp:20]: How about making handler a part of ctor, too?

File: ticker/include/ticker/Ticker.hpp [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    void stop();
+
+    [[nodiscard]] period_type delta_time() const;
+    [[nodiscard]] clock_type::time_point started_at() const;
```
Comment (radameon) [inline ticker/include/ticker/Ticker.hpp:24]: `start_time` is a better name

File: ticker/include/ticker/Ticker.hpp (hunk 1 above)
Comment (Sckorn) [reply ticker/include/ticker/Ticker.hpp]: Introduced a template ctor

File: ticker/include/ticker/Ticker.hpp (hunk 2 above)
Comment (Sckorn) [reply ticker/include/ticker/Ticker.hpp:45]: Made a handler a part of ctor, left this method intact, since it might be requested.

**Pull request merged**